lib/v2/iresearch/report.js

```diff
@@ -1,12 +1,17 @@
 const got = require('@/utils/got');
-const date = require('@/utils/date');
+const { art } = require('@/utils/render');
+const path = require('path');
+const { parseDate } = require('@/utils/parse-date');
+
 async function processReport(item, ctx) {
-    const apiUrl = `http://www.iresearch.com.cn/Detail/reportM?id=${item.NewsId}&isfree=0`;
-    const pageUrl = 'http://www.iresearch.com.cn/m/report.shtml';
+    const apiUrl = `https://www.iresearch.com.cn/api/Detail/reportM?id=${item.NewsId}&isfree=0`;
+    const pageUrl = 'https://www.iresearch.com.cn/m/report.shtml';
+
     const cache = await ctx.cache.get(apiUrl);
```

Comment:
Please also update cache access method from get/set to tryGet
docs: https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben